.azure/templates/steps/clean_maven_snapshots.yaml

```diff
@@ -0,0 +1,8 @@
+steps:
+  # Clean external SNAPSHOT dependencies before caching
+  - bash: |
+      mvn dependency:purge-local-repository \
+        -DsnapshotsOnly=true \
+        -DreResolve=false \
+        -Dexcludes="io.strimzi:*" -q || true
```

Comment:
I would prefer to not use snapshot dependencies in the first place. But assuming we want to stick with this ... what is the reason to not clean up Strimzi snapshots either?